<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Csv2Json Converter</string>
  </property>
  <widget class="QWidget" name="wMain">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="content">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cbIsRunning">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="hs2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="MDLineEdit" name="leSourcePath">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="label" stdset="0">
          <string>Source Path</string>
         </property>
         <property name="labelX" stdset="0">
          <number>14</number>
         </property>
         <property name="labelY" stdset="0">
          <number>28</number>
         </property>
         <property name="labelTextWidth" stdset="0">
          <number>12</number>
         </property>
         <property name="labelColor" stdset="0">
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
         <property name="labelFocusColor" stdset="0">
          <color alpha="87">
           <red>46</red>
           <green>125</green>
           <blue>50</blue>
          </color>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="pteEditor"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="bottom">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="hs">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbStyle">
         <property name="text">
          <string>Style</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbStart">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbStop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MDLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ui/MDLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
